src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1804,6 +1804,41 @@ public void TestLongProcessIsWorking()
             Assert.True(p.HasExited);
         }
 
+        [Fact]
```

Comment:
This test is only meant to work on Linux, right?  If so, can you please move it to the ProcessTests.Unix.cs file and also add a [PlatformSpecific(TestPlatforms.Linux)] attribute to the test?

Reply:
> This test is only meant to work on Linux, right? 

Actually should be for Linux/Mac, but it doesn't work for Mac. 

> If so, can you please move it to the ProcessTests.Unix.cs file and also add a [PlatformSpecific(TestPlatforms.Linux)] attribute to the test?

I did it, but after Dan's review I moved it https://github.com/dotnet/corefx/pull/36880#discussion_r275409187
Maybe we could keep here with `[PlatformSpecific(TestPlatforms.Any & ~TestPlatforms.OSX)] `

Reply:
We should probably have
```
[PlatformSpecific(TestPlatforms.AnyUnix)]
[ActiveIssue(XXXX, TestPlatforms.OSX)]
```
where XXXX is some separately issue you open for someone to figure out how to make this work on Mac.

Reply:
thank's forgot that `ActiveIssue` exclude plat.